### PR TITLE
Update client server version

### PR DESCRIPTION
**Pull request by znewton**
Update server version from 0.1019.0-0 to 0.1020.0 in client

### CODE REVIEW

File: examples/data-objects/codemirror/src/presence.ts [hunk 1]
```diff
@@ -36,8 +36,8 @@ class PresenceManager extends EventEmitter {
         this.presenceKey = `presence-${runtime.id}`;
 
         runtime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
-            // Only process presence keys that are not local while we are connected
-            if (message.type === this.presenceKey && !local && runtime.connected) {
+            // Only process presence keys that are not local while we are connected and have a non-null clientId
+            if (message.type === this.presenceKey && !local && runtime.connected && message.clientId) {
```
Comment (znewton) [inline examples/data-objects/codemirror/src/presence.ts:40]: `ISignalMessage.clientId` can be `null`. The only case for this is on room join, so I don't think presence needs to be emitted for that. Let me know if I'm wrong.

File: packages/drivers/routerlicious-driver/src/documentService.ts [hunk 2]
```diff
@@ -65,11 +72,19 @@ export class DocumentService implements api.IDocumentService {
...
+            {
+                Authorization: getAuthorizationTokenFromCredentials(credentials),
+            },
+        );
         const historian = new Historian(
             this.gitUrl,
             this.historianApi,
             this.disableCache,
-            credentials);
+            restWrapper);
```
Comment (znewton) [inline packages/drivers/routerlicious-driver/src/documentService.ts:87]: Breaking change detailed in [server/BREAKING.md](https://github.com/microsoft/FluidFramework/blob/main/server/BREAKING.md#historian-class-no-longer-handles-request-headers). This change maintains functionality (except `token` query param, which is no longer necessary and I will be making a PR to remove that in server/r11s/packages/services/srctenant.ts as well

File: packages/loader/container-loader/src/deltaManager.ts [hunk 3]
```diff
@@ -1426,7 +1426,7 @@ export class DeltaManager
         }
 
         // Add final ack trace.
-        if (message.traces?.length > 0) {
+        if (message.traces !== undefined && message.traces.length > 0) {
```
Comment (znewton) [inline packages/loader/container-loader/src/deltaManager.ts:1429]: TS throws error that `messages.traces.length` could be undefined

File: examples/data-objects/codemirror/src/presence.ts (hunk 1 above)
Comment (tanviraumi) [reply examples/data-objects/codemirror/src/presence.ts:40]: Makes sense. I think the type check will also fail but better to be more conservative.

**Review by tanviraumi — APPROVED** [review]

**Review by curtisman — APPROVED** [review]

**Comment by vladsud** [comment]
@znewton , mostly FYI - not sure if things changed while this PR was out, but server is already "0.1021.0", and I'm going to bump it as part of https://github.com/microsoft/FluidFramework/pull/5381 in a moment. Please let me know if there is some reason you did not want to go to 0.1021

**Pull request merged**